src/libraries/Common/tests/System/Security/Cryptography/AesGcmTests.cs

```diff
@@ -962,6 +972,18 @@ private static IEnumerable<AEADTest> GetNistTests()
                 Tag = "4ce8aff15debc1b23c50665b9c".HexToByteArray(),
             },
         };
+
+        private static void FillRandom(Span<byte> destination)
+        {
+#if NET
+            RandomNumberGenerator.Fill(destination);
+#else
+            RandomNumberGenerator generator = RandomNumberGenerator.Create();
```

Comment:
RNG is IDisposable, so either a manual call to Dispose or a using should be added.